examples/imagenet/input_pipeline.py

```diff
@@ -94,16 +98,18 @@ def _decode_and_random_crop(image_bytes, image_size):
       image_bytes,
       bbox,
       min_object_covered=0.1,
-      aspect_ratio_range=(3. / 4, 4. / 3.),
+      aspect_ratio_range=(3.0 / 4, 4.0 / 3.0),
```

Comment:
It is a bit funny these zeros are added everywhere, seems a bit unnecessary to me.

Reply:
black (and thus pyink) are mostly non-configurable